spec/person_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'date'
+require_relative '../person/person'
+
+describe Person do
```

Comment:
- [ ] Hello! 🌟 Please ensure that you include a test case for the correct_name method within the Person class. By doing so, you'll guarantee that all implemented methods are thoroughly tested, aligning perfectly with the project guidelines. 🧪 This step is essential to validate the correctness and robustness of your code. 🚀